src/test/java/com/wikia/webdriver/testcases/desktop/discussions/CategoriesTests.java

```diff
@@ -26,14 +25,16 @@
 import static com.wikia.webdriver.elements.communities.mobile.components.discussions.common.TextGenerator.createUniqueCategoryName;
 
 
-@Execute(onWikia = MobileWikis.DISCUSSIONS_1)
+@Execute(onWikia = "qadiscussions", language = "de")
 @Test(groups = "discussions-categories")
 public class CategoriesTests extends NewTestTemplate {
 
+  private static final String wikiName = "qadiscussions";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00115)